flask_security/__init__.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 """
-    flask_security
```

Comment:
is this indentation change caused by black?

Reply:
Yes, you can find it here: https://github.com/inveniosoftware/flask-security-fork/pull/66/commits/64e3757d292e4b4046ee3b50baa6e8f5c8388843
and ignored here: https://github.com/inveniosoftware/flask-security-fork/pull/66/commits/355c3e39cbb2cd6bd2bd76328ee0e7a5565877a6